group_vars/location_segen/networks.yml

```diff
@@ -0,0 +1,188 @@
+---
+# IPv4 10.31.6.0/24
+# IPv4 10.31.66.0/24
+# IPv6 2001:bf7:760:1000::/56
+
+# DHCP: 10.31.66.0/24
+# MGMT: 10.31.6.0/26
+# MESH: 10.31.6.64/26
+
+ipv6_prefix: "2001:bf7:760:1000::/56"
+
+networks:
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.6.0/26
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      segen-core: 1
+      segen-switch: 2
+
+      # bbb links
+      segen-saarbr: 10
+      segen-f2a: 11
+      segen-mabb: 12
+      segen-sw: 13
+      segen-bht: 14
+      segen-k12: 15
+      segen-no: 16
+      segen-wsee: 17
+      
+      # airos
+      segen-n-5ghz: 20
+      segen-o-5ghz: 21
+      segen-s-5ghz: 22
+      segen-w-5ghz: 23
+
+      # nf
+      segen-n-nf-2ghz: 24
+      segen-o-nf-2ghz: 25
+      segen-s-nf-2ghz: 26
+      segen-w-nf-2ghz: 27
+
+  - vid: 10
+    role: dhcp
+    prefix: 10.31.66.0/24
```

Comment:
/26  or even /27 should be sufficient, given the maximum simultanous clients were 10?

Reply:
Sometimes there are hoffests at the church.  Also, Segen has had this address range since day 1.

```
-         10.31.6.0/24                          R  -   None               None      freifunk@faust2k.net  Segenskirche                                                                      
-       10.36.0.0/16                            R  -   None               None      None                  HNA                                                                               
-         10.36.66.0/24                         R  -   None               None      freifunk@faust2k.net  Segenskirche                                                                      
```

I would prefer to leave it how it is.

Reply:
I respectfully disagree

Reply:
see comment above regarding already assigned IPs.